test/fixtures/mongo_keyfile

```diff
@@ -0,0 +1,16 @@
+PUQGGGC+g1sxoYzKKnX7531pos0WvDCYlx1Ak5ZupX3KvyuU2iVcC9puJJSG06VR
```

Comment:
this is safe to commit right?

Reply:
Yes. This is not our key file, it's the key file we use to connect to the test mongodb replica set which always starts empty.